com.ktechpit.wonderwall.yaml

```diff
@@ -0,0 +1,42 @@
+app-id: com.ktechpit.wonderwall
+base: io.qt.qtwebkit.BaseApp
+base-version: '5.12'
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: wonderwall
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-download
+  - --device=dri
+  
+  - --talk-name=org.freedesktop.FileManager1
+  
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  
+  - --talk-name=org.elementary.Contractor
+  - --system-talk-name=org.freedesktop.Accounts
+  
+modules:
+  - name: ww
+    buildsystem: qmake
+    config-opts:
+      - QMAKE_STRIP=
+      - DEFINES+=USE_DBUS
+      - INCLUDEPATH+="${FLATPAK_DEST}/include/QtWebKitWidgets"
+      - INCLUDEPATH+="${FLATPAK_DEST}/include/QtWebKit"
+      - LRELEASE=lrelease
+    build-options:
+      env:
+        QMAKEPATH: "/app/lib"
+    sources:
+      - type: archive
+        url: http://ktechpit.com/USS/WW/dist/wonderwall_7.0.388_flathub.zip
```

Comment:
since you didn't reply on matrix, you know this give anyone access to the source code...